napari/utils/perf/__init__.py

```diff
@@ -1,41 +1,42 @@
-"""Performance Monitoring init.
+"""Performance Monitoring.
 
-USE_PERFMON is true only if NAPARI_PERFMON environment variable is set and not
-zero.
+To enable perfmon define the env var NAPARI_PERFMON to be non-zero.
```

Comment:
I think it would be nice to add a bit more detail on what is meant by "non-zero". Perhaps something like:

```suggestion
To enable perfmon define the env var NAPARI_PERFMON to be non-zero. For example:

- set to 1 to activate perfmon and use the menu Debug -> Performance Trace in the napari GUI
- set to the path to your perfmon config file to configure napari to start up with perfmon set up as specified in the config file. See below for more information of the perfmon config file.
```

Reply:
I thought that was covered somewhere, buy couldn't find here. I redid that whole comment block just now and including your ideas. Seems better now.

Reply:
Awesome! Thanks!